frontend/src/components/profile/directories/DirectoriesSection.tsx

```diff
@@ -237,7 +258,8 @@ const DirectorySection = ({
                 )}
 
                 <DataGridPro
-                    autoHeight
```

Comment:
I think we should keep the `autoHeight` parameter for now. I know it is deprecated, but for now I don't want to change that. I'll eventually go through and migrate all the usages of `autoHeight`